src/routes/auth/decorators/auth.decorator.spec.ts

```diff
@@ -47,11 +53,28 @@ describe('PaginationDataDecorator', () => {
   beforeEach(async () => {
     jest.resetAllMocks();
 
+    const baseConfiguration = configuration();
+    const testConfiguration = (): typeof baseConfiguration => ({
+      ...baseConfiguration,
+      features: {
+        ...baseConfiguration.features,
+        auth: true,
+      },
+    });
+
     const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [TestModule],
+      imports: [
+        TestModule,
+        TestLoggingModule,
+        ConfigurationModule.register(testConfiguration),
+        CacheModule,
+        TestNetworkModule,
+      ],
```

Comment:
This change was necessary because of the inclusion of `SafeRepository` for owner checking.

Reply:
Isn't that specific to the email feature? The `AuthGuard` by itself shouldn't need it

Reply:
True. This was an oversight from the migration of ownership checking. Removed in 72f78b8985d92a7279ab99aafc3de01e84ecfdd7.